File: Sorting/Selection Sort/SelectionSort.php
<?php
    //below is a function for selction sort
    function selectionSort($arr,$n){
        for($i=0;$i<$n;$i++){
          //our aim is to find the minimum value in the array and
          //then put it in front till we get our array sorted
            $min=$i;
            for($j=$i;$j<$n;$j++){
                if($arr[$min]>$arr[$j]){
                    $min=$j;
                }
            }
            $tmp= $arr[$min];
            $arr[$min]= $arr[$i];
            $arr[$i]= $tmp;
        }
        return $arr;
    }
    //we want to know the length of array
    $n = readline("Enter the length of array- ");
    //now declearing an empty array
    echo("Enter the values of array separated by space\n");
    //we are storing data in an array
    //explode breaks string into array
    $arr = explode(' ', readline()); 
    $arr= selectionSort($arr,$n);
    print_r($arr);
?> 

File: Sorting/Selection Sort/SelectionSort.cpp
/*
	*Select* minimum in an array and put it in front of array
	Selection Sort

	input format:
		first enter size of array
		then array

		eg.

		5
		5 4 3 2 1
*/
#include <iostream>
using namespace std;

void SelectionSort(int *arr, int size) {
	for (int i = 0; i < size; i++) {
		int min = i;                              //min will contain the index of minimum value element
		for (int j = i; j < size; j++) {          //this loop will find the min position
			if (arr[min] > arr[j])
				min = j;
		}

		//swap min position with ith
		int temp = arr[min];
		arr[min] = arr[i];
		arr[i] = temp;
	}
}

int main() {
	int size;
	cin >> size;

	int arr[size];
	for (int i = 0; i < size; i++)
		cin >> arr[i];

	SelectionSort(arr, size);

	cout << "Sorted array: \n";
	for (int i = 0; i < size; i++)
		cout << arr[i] << " ";
	cout << endl;

	return 0;
}